### ISSUE TITLE
PHP errors not logged via stdout/stderr

### ISSUE THREAD

**Issue by Hackebein** [richarvey/nginx-php-fpm#239]
PHP errors are not visible via `docker logs`
Tested Versions:
* 1.3.8
* 1.3.10

`test.php`:
``` PHP
<?php

test_notexists();
```
`docker logs`:
```
172.18.0.6 - - [07/May/2018:20:31:00 +0200] "GET /test.php HTTP/1.1" 200 215 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36"
```

After enable file logs:
```
/var/www/html # tail -f /var/log/fpm-php.www.log
[07-May-2018 20:43:47 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function test_notexists() in /var/www/html/test.php:3
Stack trace:
#0 {main}
  thrown in /var/www/html/test.php on line 3
```

Started via Docker Compose:
```
  website:
    image: richarvey/nginx-php-fpm:1.3.10
    volumes:
     - /etc/localtime:/etc/localtime:ro
     - ./htdocs:/var/www/html:rw
    restart: unless-stopped
    ports:
     - 80:80
```

**Comment by apanagiotou** [comment]
+1

**Comment by jbouzekri** [comment]
I am not sure if it is related but I have just sumitted this SO issue : https://stackoverflow.com/questions/50995042/docker-does-not-catch-php-fpm-outputs-with-symfony-and-monolog

**Comment by atefBenSalah** [comment]
+1

**Comment by richarvey** [comment]
fixed in 1.5.4
